### PR TITLE
Fixed a Broken Link in 3.6 Quantum Phase Estimation

### PR DESCRIPTION
**Pull request by divshacker**
# Changes made

This PR is clearly the fix of #1001. All details can be found in this Issue. I have changed the name of the jupyter file so that it would work as expected. 
# Justification

### CODE REVIEW

File: content/ch-applications/vqe-molecules.ipynb [hunk 1]
```diff
@@ -361,7 +361,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Assume the depth setting is set to $d$. Then, RyRz has $n\\times (d+1)\\times 2$ parameters, Ry with linear entanglement has $2n\\times(d + \\frac{1}{2})$ parameters, and Ry with full entanglement has $d\\times n\\times \\frac{(n + 1)}{2} + n$ parameters."
+    "Assume the depth setting is set to $d$. Then, RyRz has $n\\times (d+1)\\times 2$ parameters, Ry with linear entanglement has $n\\times (d+1)$ parameters, and Ry with full entanglement has $n\\times (d+1)$ parameters."
```
Comment (frankharkins) [inline content/ch-applications/vqe-molecules.ipynb:364]: Not sure if this is intentional, but if this is a suggested change, can you make it in another PR please?

File: content/ch-algorithms/quantum-phase-estimation.ipynb [hunk 2]
```diff
@@ -115,7 +115,7 @@
     "\\end{aligned}\n",
     "where $k$ denotes the integer representation of n-bit binary numbers. \n",
     "\n",
-    "iv. **Inverse Fourier Transform**: Notice that the above expression is exactly the result of applying a quantum Fourier transform as we derived in the notebook on [Quantum Fourier Transform and its Qiskit Implementation](qft.ipynb). Recall that QFT maps an n-qubit input state $\\vert x\\rangle$ into an output as\n",
+    "iv. **Inverse Fourier Transform**: Notice that the above expression is exactly the result of applying a quantum Fourier transform as we derived in the notebook on [Quantum Fourier Transform and its Qiskit Implementation](quantum-fourier-transform.ipynb). Recall that QFT maps an n-qubit input state $\\vert x\\rangle$ into an output as\n",
```
Comment (frankharkins) [inline content/ch-algorithms/quantum-phase-estimation.ipynb:118]: Thanks for this, we generally link to the 
```suggestion
    "iv. **Inverse Fourier Transform**: Notice that the above expression is exactly the result of applying a quantum Fourier transform as we derived in the notebook on [Quantum Fourier Transform and its Qiskit Implementation](https://qiskit.org/textbook/ch-algorithms/quantum-fourier-transform.html). Recall that QFT maps an n-qubit input state $\\vert x\\rangle$ into an output as\n",
```
Thanks for doing this, we generally link to the live site so the links work both as local notebooks and also on qiskit.org/textbook. Not a perfect solution but seems to work fine for now.

**Review by frankharkins — CHANGES_REQUESTED** [review]

File: content/ch-applications/vqe-molecules.ipynb (hunk 1 above)
Comment (divshacker) [reply content/ch-applications/vqe-molecules.ipynb:364]: Actually I am trying to do that same but don't know how XD

File: content/ch-algorithms/quantum-phase-estimation.ipynb (hunk 2 above)
Comment (divshacker) [reply content/ch-algorithms/quantum-phase-estimation.ipynb:118]: Yes. Thanks for clarifying that. I previously contributed to qiskit documentation and there only local notebook path are needed so I thought. But from now I will try to add site links only.

File: content/ch-applications/vqe-molecules.ipynb (hunk 1 above)
Comment (divshacker) [reply content/ch-applications/vqe-molecules.ipynb]: I have again changed that to what it was before. The best I can do now

**Comment by divshacker** [comment]
I have done all things as you suggested. @frankharkins . 
Let me know if any further changes needed.
Thank you

**Review by frankharkins — APPROVED** [review]
Looks good, thank you!
